heroic-core/pom.xml

```diff
@@ -49,8 +49,16 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <version>27.0-jre</version>
```

Comment:
all the versions should be controlled in the parents pom dependendency management section

Reply:
https://github.com/spotify/heroic/blob/master/pom.xml#L545